src/test/java/fi/aalto/cs/apluscourses/ui/CheckBoxTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
+import fi.aalto.cs.apluscourses.ui.base.CheckBox;
```

Comment:
I think the test should be in the `base` package instead of importing `CheckBox` here

Reply:
You're correct, of course.